common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/Indexer.scala

```diff
@@ -68,25 +68,35 @@ trait Indexable[T] {
   *   became an issue we'd probably want to do something like:
   *
   *   def version(image: AugmentedImage) =
-  *     10 * (2*image.version + image.source.version)
+  *     versionMultiplier * (2*image.version + image.source.version)
   */
 object Indexable extends Logging {
 
+  final val versionMultiplier = 1000
+
   implicit val imageIndexable: Indexable[AugmentedImage] =
     new Indexable[AugmentedImage] {
       def id(image: AugmentedImage) =
         image.id.canonicalId
 
       def version(image: AugmentedImage) =
-        10 * (image.version + image.source.version) +
+        versionMultiplier * (image.version + image.source.version) +
           (image.source match {
-            case SourceWorks(_, _, nMergedSources) if nMergedSources >= 10 =>
-              throw new RuntimeException(
-                s"Image ${image.id.canonicalId} has $nMergedSources >= 10 merged sources; versioning/ingest may be inconsistent")
-            case SourceWorks(_, Some(redirected), nMergedSources) =>
-              nMergedSources + 1
-            case SourceWorks(_, None, nMergedSources) => nMergedSources
+            case SourceWorks(_, _, nSources) if nSources >= versionMultiplier =>
+              warnOnTooManySources(image, nSources)
+              // Clamp the increment so as not to override higher versions
+              versionMultiplier - 1
+            case SourceWorks(_, Some(redirected), 1)        => 1
+            case SourceWorks(_, Some(redirected), nSources) => nSources
+            case SourceWorks(_, None, 1)                    => 0
+            case SourceWorks(_, None, nSources)             => nSources - 1
```

Comment:
Yes it is, I changed it from this because I found it confusing (in my mind, the spirit of line 90 is really `(nSources - 1) + 1` but I think I've changed my mind again and will change it back